protocol/xmrmaker/instance_test.go

```diff
@@ -204,6 +223,8 @@ func TestInstance_createOngoingSwap(t *testing.T) {
 }
 
 func TestInstance_CompleteSwap(t *testing.T) {
+	_ = logging.SetLogLevel("protocol", "debug")
+	_ = logging.SetLogLevel("xmrmaker", "debug")
```

Comment:
These might be more helpful at the test package level. Your call. I've been putting them in `init` methods at the test package level.  I'm slightly tempted to put the function that sets them all in `common`, and then have `main` and the tests use the same function to enable logging levels.

Reply:
ah I just put these here to debug, will remove